_posts/2024-04-04

```diff
@@ -0,0 +1,7 @@
+---
+title: A bioinformatician's guide to serovars and antigenic formulae in Salmonella
+external: https://www.happykhan.com/posts/binfie-guide-serovar/
+---
+
+In the post, I will describe the basics of Salmonella serovar nomenclature. This post will act as a primer for bioinformaticians starting to work with Salmonella genomes, who are often at a loss.
```

Comment:
In the post, I will describe the basics of *Salmonella* serovar nomenclature. This post will act as a primer for bioinformaticians starting to work with *Salmonella* genomes, who are often at a loss.

Reply:
Please consider using the ["Suggestion Mode" feature of GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) (see step 6). 

By providing a suggestion using the proper suggestion mode:

1. For authors, it is **unambiguous** what you are proposing 
2. It's *also* easier for them to simply accept the suggestion, PR authors prefer suggestions!
3. You get credited in the Git commit helping us properly track attribution